src/content/sellers-guide/index.mdx

```diff
@@ -40,14 +40,21 @@ Initial proposal submissions: Closed on October 20, 2023.
         img_src="images/sellers/have-a-contract-image.svg"
         img_alt="Person standing amidst a series of charts and graphs."    
     >
-        ### [Confirm you’re registered in Symphony and administer your master contract](/sellers-guide/register-in-symphony/)  
-        Authorized employees must register in Symphony to complete administrative duties; check out the registration instructions to help you through the process.
+        ### [Register in FedConnect to manage your contract - FedConnect.net](https://www.fedconnect.net/FedConnect/Default.htm)
+        - Add “[CALM@unisonglobal.com](CALM@unisonglobal.com)” to your trusted address book so messages from your GSA Contracting team do not go to spam. 
```

Comment:
currently the link cause 404 error, email link needs to add "mailto:",  e.g.`[CALM@unisonglobal.com](mailto:CALM@unisonglobal.com)`

Reply:
thanks... added